src/main/webapp/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Index</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="navigation">
+      <a href="main_hub.html" id="hub">Main Hub</a>
```

Comment:
are index and main hub the same thing? or is the index page something else

Reply:
I guess it could be the same thing. I just created it so that when the dev server is run, there's a page that exists when it's first opened.

Reply:
Cool that works! This could potentially be a little description of what our website does/ an opportunity to login.

Reply:
index.html, index.php, index.jsp, etc.. server fetches them automatically if you navigate into a directory without specifying an actual page.  ie. if you go "mysite.com/something" it would actually take you to "mysite.com/something/index.html" even if the page name is not explicitly specified